llvm/include/llvm/CodeGen/GlobalISel/CombinerHelper.h

```diff
@@ -963,6 +963,9 @@ class CombinerHelper {
 
   // Simplify (cmp cc0 x, y) (&& or ||) (cmp cc1 x, y) -> cmp cc2 x, y.
   bool tryFoldLogicOfFCmps(GLogicalBinOp *Logic, BuildFnTy &MatchInfo);
+
+  // Simplify (trunc v1) && (trunc v2) -> trunc (v1 && v2)
```

Comment:
`&&` seems wrong here?
```suggestion
  // Simplify (trunc v1) & (trunc v2) -> trunc (v1 & v2)
```